### ISSUE TITLE
Suggestion: change 10s to 30s or 60s (or detect menus open)

### ISSUE THREAD

**Issue by mjbaldwin** [Mortennn/Dozer#151]
First all of, love using the app! It helps so much.

But I've encountered that many times, I'll expand my icons to use one of my menubar tools with a menu of its own, and while I'm looking for the right submenu option or something, the whole thing closes -- because it autocloses after 10s.

Now I really like the autoclose option generally. I tried leaving it off, but never remembered to close the expanded menubar, so I really do need it.

But 10s is definitely too short. Ideally you could detect that I've got a menu open on one of my items, and never close while that's the case. But since I assume that's not possible or reliable, I'd like to suggest changing to either 30s or 60s. (Or provide a dropdown to select between 10/30/60).

Again, GREAT app!

**Comment by ioemilio** [comment]
I agree that also 30 seconds (current maximum) be a bit short in some situations.

**Comment by mjbaldwin** [comment]
How do you change the value? Where is there a maximum? In my settings dialog the 10 seconds appears to be hard-coded:

![image](https://user-images.githubusercontent.com/160959/109191426-28ceab80-7764-11eb-8160-844804395208.png)

**Comment by ioemilio** [comment]
In version 4.2.0 you have a menu with multiple choices:

**Comment by mjbaldwin** [comment]
Oh thanks I hadn't realized! Downloading now.

Also the "use full width" feature looks amazing, something I never even thought to ask for. What a great idea.
